specification/iotcentral/data-plane/Microsoft.IoTCentral/preview/2019-10-28-preview/iotcentral.json

```diff
@@ -1278,9 +1642,6 @@
               "$ref": "#/definitions/DeviceCommandCollection"
             }
           }
-        },
-        "x-ms-pageable": {
```

Comment:
Why do you remove this? For what I see "DeviceCommandCollection" has nextLink/value as expected for a pageable operation. Seems inconsistent to me.